Taskfile.yml

```diff
@@ -324,56 +322,50 @@ tasks:
       - pre-commit run -a
 
   run-website:
-    desc: Build & serve the static website.
+    desc: Build & serve the static website for interactive development.
     dir: web/website
     cmds:
       - hugo server
 
-  run-web:
-    desc: Build & serve the website & playground.
-    summary:
-      Note that this only live-reloads the static website; and requires
-      rerunning to pick up playground & book changes.
-    dir: web/website
-    cmds:
-      - task: build-web
-      # Then start web server with rendering to disk, so it picks up the playground files.
-      - hugo server --renderToDisk
-
   run-book:
-    desc: Build & serve the book.
+    desc: Build & serve the book for interactive development.
     dir: web/book
     cmds:
       - mdbook serve --port=3000
 
   run-playground:
-    desc: Build & serve the playground.
-    dir: web/playground
-    cmds:
-      - npm install
-      - npm start
-
-  run-playground-cached:
-    desc: Build & serve the playground, without rebuilding rust code.
+    desc: Build & serve the playground for interactive development.
     dir: web/playground
     cmds:
-      - task: install-playground-npm-dependencies
+      - task update-playground-dependencies
       - npm start
 
-  install-playground-npm-dependencies:
-    # Look to see if npm dependencies need to be updated
-    # Use task's sources/generates to see if checksums of
-    # node_modules directory have changed since package.json was updated
-
-    desc: Check to see if package.json has changed since we ran npm install
+  update-playground-dependencies:
+    # Check if npm dependencies for the playground need to be updated
+    # Use task's sources/generates to see if package.json,
+    # or anything in crates or bindings was updated after the
+    # node_modules was rebuilt
+    desc: Install dep's if something important changed recently
     dir: web/playground
     cmds:
-      - npm install
+      - task install-playground-dependencies-all
     sources:
       - package.json
+      - ../../crates/**/*.rs
+      - ../../bindings/**/*.rs
     generates:
       - node_modules/*
 
+  install-playground-dependencies-all:
+    # `npm install --install-links=false` no longer necessary
+    # Formerly it was used to install prql-js
+    # as the regular dependency, instead of creating a
+    # symlink. Refer to https://github.com/PRQL/prql/pull/1296.
```

Comment:
I'm not sure we really need this now — but OK if we leave it for a while and then remove later

Reply:
I'll leave the first line of that comment as a reminder and remove the other 3 lines